src/live_view.cpp

```diff
@@ -1,38 +1,68 @@
 #include "live_view.h"
 #include "output.h"
 #include "game.h"
+#include "map.h"
 #include "options.h"
+#include "translations.h"
 
-#define START_LINE 1
-#define START_COLUMN 1
+#include <map>
+#include <string>
 
-live_view::live_view() : compact_view(false), w_live_view(NULL),
-    enabled(false), inuse(false), last_height(-1)
+namespace {
+constexpr int START_LINE = 1;
+constexpr int START_COLUMN = 1;
+
+void print_items(WINDOW *const w, const map_stack &items, int &line)
```

Comment:
Not your fault, and I'd prefer to keep this a bugfix anyway, but I'm pretty sure there's another piece of code that does the exact same thing elsewhere.